requirements.txt

```diff
@@ -38,7 +38,7 @@ py==1.10.0
 pyasn1==0.4.4
 pycparser==2.19
 Pygments==2.14.0
-pylint>=2.14.5
+pylint<=2.17.0-dev0,>=2.15.4
```

Comment:
I'm guessing this is the range returned by the error log. Should we be using the range or just >=2.15.4?